### PR TITLE
Create setup.py for beaker Python package

### PR DESCRIPTION
**Pull request by gnestor**
Closes https://github.com/twosigma/beakerx/issues/5156 and https://github.com/twosigma/beakerx/issues/5067

This allows the beaker Python package to be installed using pip from the root directory of the project: `pip install -e .`

Once installed, `beaker` can be imported from any directory: `from beaker.plot import *`

### CODE REVIEW

**Comment by gnestor** [comment]
The beaker package is now importable, but the `Plot` class doesn't seem to be working properly:

```py
from beaker.plot import Plot

Plot(title="Title",
     xLabel="Horizontal",
     yLabel="Vertical",
     initWidth=500,
     initHeight=200)
```

image

File: beaker/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from .runtime import *
```
Comment (scottdraves) [inline beaker/__init__.py:15]: all files need banner

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+setup_args = dict(
+    name                    = 'beaker',
+    description             = 'BeakerX runtime for Python',
+    long_description        = 'BeakerX runtime for Python',
+    version                 = '0.1.0',
+    author                  = 'Two Sigma Open Source, LLC',
+    author_email            = 'beaker-feedback@twosigma.com',
+    url                     = 'http://beakernotebook.com',
+    license                 = 'BSD',
```
Comment (scottdraves) [inline setup.py]: apache2

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from setuptools import setup
```
Comment (scottdraves) [inline setup.py:15]: banner

**Review by scottdraves — CHANGES_REQUESTED** [review]
looks good, just a few details.
plus, please update README with install instructions.

**Comment by gnestor** [comment]
Ok, updated!

**Comment by gnestor** [comment]
Is there a display class that needs to be used with `Plot`?

**Pull request merged**